packages/inventory-compliance/src/Utilities/RulesHelper.js

```diff
@@ -1,15 +1,14 @@
-export const toRulesArray = (policiesWithRules) => {
-    const rules = policiesWithRules.flatMap((policy) => (
-        policy.rules.map((rule) => {
+export const toRulesArray = (profileRules) => (
```

Comment:
It is passed an array if profiles with an array of rules, so, `ruleProfiles` would be more appropriate here.

Reply:
As discussed in slack, I'd prefer just naming this `profilesWithRules` or simply `profiles`. I really don't like `profileRule` being a profile and not a rule.

Reply:
Okay, I've composed it into `profilesWithRules > profileWithRules > profile/rules`.